presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -440,6 +442,7 @@ protected Type toPrestoType(int jdbcType, int columnSize)
                 return BIGINT;
             case Types.FLOAT:
             case Types.REAL:
```

Comment:
Hmm if something is type FLOAT it should return DOUBLE. (I know it's weird, but that's just how the JDBC spec is)